src/news/news.js

```diff
@@ -0,0 +1,51 @@
+import './news.scss';
```

Comment:
From the naming perspective, I would propose to name component files from the capital letter (News.js)

Reply:
done